pkg/topology/depthmonitor/depthmonitor.go

```diff
@@ -156,6 +157,16 @@ func (s *Service) manage(warmupTime, wakeupInterval time.Duration) {
 			}
 		}
 	}
+	next()
+
+	for {
+		select {
+		case <-s.quit:
+			return
+		case <-time.After(wakeupInterval):
```

Comment:
I'd recommend using `NewTimer` for efficiency. The documentation says:
> The underlying Timer is not recovered by the garbage collector until the timer fires. If efficiency is a concern, use NewTimer instead and call Timer.Stop if the timer is no longer needed.

Reply:
efficiency is not a concern though, at least in this context :)